geometry_msgs/PoseStamped[] poses
bool[] allow_skip
float32 cell_width
